documentation/docs/developer_guide/coding_standards.md

```diff
@@ -1 +1,160 @@
+ * Specify IMPLICIT NONE 
+ * All IF constructs must be of the form:
+```
+IF() THEN 
+  Even if they only contain a simple, single line statement following the IF. 
+END IF
+```
+ * In summary, be pedantic, be clear
+
+## Final Note about JAC
+The UKMO is in the process of revising their entire suite of models, as part of a major program to create a Next Generation Modelling System (NGMS). 
+The specific coding standards required of the NGMS have not yet been strictly defined. 
+However, from a technical perspective the necessity of the project can be summarized as follows. 
+Excluding a quantum computing breakthrough, we as a community can no longer expect advances in hardware to enable higher performance. This means higher resolution, more sophisticated process description etc. We can however better  organize the software. 
+In the main our models (including CABLE), have evolved over decades, involving dozens of developers. A key feature emerging from NGMS is the tight reign on memory. As such it will not be permitted to USE data through MODULEs. Data must be passed through argument lists. There are further rules emerging in this respect which relax the rules slightly to allow time independent scalars to be USEd. Nevertheless it is straight-forward enough to pass these as well. 
+
+## Specific Develoments
+Code developments broadly fall into two categories. Bug fixes and new developments. Both can of course involve a wide range of complexity. However, at the extremes a bug fix may be as simple as moving a bracket. Alternatively, the bug fix might require you to rewrite a section, or even several sections of code. 
+A new development might be as simple as an alternative section or calculation, or it might be an entirely new model that can be plugged in to CABLE. Either as an alternative model, or an extra feature to be used in the standard model.
+
+In all cases an issue should be raised. The accompanying fix contained should then be implemented in a unique branch, corresponding to the raised issue. A bugfix might simply ivolve reference to this issue. For a more complicated, added feature, it is a requirement that this addition can be isolated, switched off via a configuration switch. So that the model can be run, and return results as if there had been no addition. There is of course every possibility that this feature will be accepted as a standard part of the model. Although this is not the expectation initially and the developer should be mindful of this fundamental requirement. 
+
+We proceed assuming that the "issue" raised requires adding a new and significant feature to CABLE, requiring the addition of new files/modules to CABLE. Lesser modifications are covered under this assumption anyway.
+
+For discussion purposes we assume that you are including a new model to do some new stuff. This can conveniently be added in a single file. 
+
+## Code Template 
+Assuming the *new* model is called from eg_driver.F90.
+
+```
```

Comment:
```suggestion
```fortran
```

Reply:
Do you mean just adding the word "fortran"? Do you think this makes it clearer that a bunch of statements go here?

Reply:
No i mean this is markdown for specifying the language of the code block. it add fortran syntax highlighting.

Reply:
cool feature. I'll add it